src/frontend/src/components/authenticator.js

```diff
@@ -21,6 +22,11 @@ export default function Authenticator(Comp) {
       this.props.loadProfile();
     }
 
+    componentDidMount() {
+      const props = this.props;
+      props.store.dispatch(getNeighborhoods());
```

Comment:
This used to be called in the `setAuthToken` action, which happens right in this component render method. So I assumed that listing neighborhood should happen when this component mounts, since the related endpoints are not behind auth anymore after the change on the backend.

Reply:
Good call! Now that we don't rely on having the authentication token, we can load the neighborhoods whenever it's most convenient, and this does seem like a better spot.